gemfire-core/src/main/java/com/gemstone/gemfire/internal/cache/versions/RegionVersionVector.java

```diff
@@ -669,6 +669,11 @@ public void initializeVersionHolder(T mbr, RegionVersionHolder<T> otherHolder) {
         this.localExceptions.initializeFrom(vh);
       }
     } else {
+      if (mbr.equals(this.myId)) {
+        RegionVersionHolder<T> vh = otherHolder;
+        long version = vh.version;
+        updateLocalVersion(version);
+      }
```

Comment:
Actually, h and this.localExceptions refer to the same object.
While recording the version we record in localExceptions and put it into memberToVersion map.